exercises/accumulate/test/accumulate_test.exs

```diff
@@ -1,7 +1,7 @@
-if System.get_env("EXERCISM_TEST_EXAMPLES") do
-  ExUnit.start()
-  ExUnit.configure(exclude: :pending, trace: true)
-end
+# if System.get_env("EXERCISM_TEST_EXAMPLES") do
+#   ExUnit.start()
+#   ExUnit.configure(exclude: :pending, trace: true)
+# end
```

Comment:
Since these are in git history, I don't see any reason not to delete them, especially as their presence could confuse students.

Reply:
See above, agreed